configs/fsaf/fsaf_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,47 @@
+_base_ = '../retinanet/retinanet_r50_fpn_1x_coco.py'
+# model settings
+model = dict(
+    type='FSAF',
+    bbox_head=dict(
+        type='FSAFHead',
+        num_classes=80,
+        in_channels=256,
+        stacked_convs=4,
+        feat_channels=256,
+        reg_decoded_bbox=True,
+        anchor_generator=dict(
+            type='AnchorGenerator',
+            octave_base_scale=1,
+            scales_per_octave=1,
+            ratios=[1.0],
+            strides=[8, 16, 32, 64, 128]),
```

Comment:
The `AnchorGenerator` here could be confusing since we stated that only anchor free branch is implemented.  We may add comments saying that this anchor generator is only of **1** level, which is actually an instance map.